src/components/MapHexTooltip/MapHexTooltip.tsx

```diff
@@ -47,8 +48,8 @@ export const MapHexTooltip = ({
 
       <div className={s.labels}>
         <div className={s.column}>
-          <span>{axis.x.label}</span>
-          <span>{axis.y.label}</span>
+          <span>{axis.x.label || formatBivariateAxisLabel(axis.x.quotients)}</span>
+          <span>{axis.y.label || formatBivariateAxisLabel(axis.y.quotients)}</span>
```

Comment:
```suggestion
          <span>{axis.x.label ?? formatBivariateAxisLabel(axis.x.quotients)}</span>
          <span>{axis.y.label ?? formatBivariateAxisLabel(axis.y.quotients)}</span>
```

Reply:
it comes as an empty string from backend, so it won't work this way 🤷‍♂️